clients/extension/scripts/popup.js

```diff
@@ -80,32 +82,36 @@ function find(query, context = document) {
 
 /**
  * @param {Error} err
- * @return {HTMLAnchorElement}
+ * @return {HTMLButtonElement}
  */
 function buildReportErrorLink(err) {
   const issueBody = `
 **Lighthouse Version**: ${getLighthouseVersion()}
 **Lighthouse Commit**: ${getLighthouseCommitHash()}
 **Chrome Version**: ${getChromeVersion()}
 **Initial URL**: ${siteURL}
-**Error Message**: ${err.message}
+**Error Message**: ${err.friendlyMessage || err.message}
 **Stack Trace**:
 \`\`\`
 ${err.stack}
 \`\`\`
     `;
 
-  const url = new URL('https://github.com/GoogleChrome/lighthouse/issues/new');
-
-  const errorTitle = err.message.substring(0, MAX_ISSUE_ERROR_LENGTH);
-  url.searchParams.append('title', `Extension Error: ${errorTitle}`);
-  url.searchParams.append('body', issueBody.trim());
-
-  const reportErrorEl = document.createElement('a');
+  const reportErrorEl = document.createElement('button');
   reportErrorEl.className = 'button button--report-error';
-  reportErrorEl.href = url.href;
-  reportErrorEl.textContent = 'Report Error';
-  reportErrorEl.target = '_blank';
+  reportErrorEl.textContent = 'Copy to clipboard';
+
+  reportErrorEl.addEventListener('click', async () => {
+    try {
+      // @ts-ignore
+      await navigator.clipboard.writeText(issueBody);
+      reportErrorEl.textContent = 'Copied to clipboard';
+      setTimeout(() => {
+        reportErrorEl.textContent = 'Copy to clipboard';
```

Comment:
should pull this string out to a variable in a higher scope so it can be shared between this and the original setting on line 102

Reply:
Fixed in https://github.com/GoogleChrome/lighthouse/pull/8294/commits/84bac693c0ce0d722bd3bf2fa95838afcab258b2